web/jobs/jobs.go

```diff
@@ -199,6 +201,14 @@ func newTrigger(c echo.Context) error {
 		}
 	}
 
+	// Handle metadata
+	md := metadata.New()
+	if req.Metadata == nil {
+		req.Metadata = md
+	} else {
+		req.Metadata.EnsureCreatedFields(md)
+	}
```

Comment:
I don't see why we are using allowing the apps to give some metadata, and we are not using the JWT to create a metadata object with verified information